### PR TITLE
Main Menu + .json File Handling For Questions

### PR DESCRIPTION
**Pull request by Satyamjyotsanagargee**

### CODE REVIEW

**Review by dwaipayan05 — COMMENTED** [review]
Everything Looks Good, few changes to be made in the Credits Page later!  Look out for the other reviews

File: local.properties [hunk 1]
```diff
@@ -4,5 +4,5 @@
 # Location of the SDK. This is only used by Gradle.
 # For customization when using a Version Control System, please read the
 # header note.
```
Comment (prakritigoyal19) [inline local.properties:6]: Please do not commit this file

File: .project [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (prakritigoyal19) [inline .project:1]: Usually the .* files are hidden files and most of the time they are not meant to be committed. Please do not commit this file.

File: .settings/org.eclipse.buildship.core.prefs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+arguments=
```
Comment (prakritigoyal19) [inline .settings/org.eclipse.buildship.core.prefs:1]: Please do not commit this file.

File: .vscode/launch.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (prakritigoyal19) [inline .vscode/launch.json:1]: Please do not commit this file.

File: app/src/main/java/com/example/ieapp/HomeActivity.java [hunk 5]
```diff
@@ -30,39 +30,41 @@ public void play(View view)
...
+                startActivity(i);
+
+            }
+        });
+        b = (ImageButton) findViewById(R.id.button4);
         b.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                startActivity(new Intent(getApplicationContext(),Credits.class));
+                startActivity(new Intent(getApplicationContext(), Credits.class));
```
Comment (prakritigoyal19) [inline app/src/main/java/com/example/ieapp/HomeActivity.java:81]: These changes, could maybe be done the first time you write the functions. They could be made in the previous commit itself.

File: app/src/main/java/com/example/ieapp/HomeActivity.java [hunk 6]
```diff
@@ -30,39 +30,41 @@ public void play(View view)
...
             }
         });
 
 
-        play=(Button) findViewById(R.id.button4_play);
-        ring= MediaPlayer.create(HomeActivity.this,R.raw.games_of_part_4);
+        play = (Button) findViewById(R.id.button4_play);
```
Comment (prakritigoyal19) [inline app/src/main/java/com/example/ieapp/HomeActivity.java]: This change could be added in the previous commit.

File: app/src/main/java/com/example/ieapp/HomeActivity.java [hunk 7]
```diff
@@ -30,39 +30,41 @@ public void play(View view)
...
+        ring = MediaPlayer.create(HomeActivity.this, R.raw.games_of_part_4);
```
Comment (prakritigoyal19) [inline app/src/main/java/com/example/ieapp/HomeActivity.java]: This change could be added in the previous commit.

**Review by prakritigoyal19 — CHANGES_REQUESTED** [review]
While submitting the pr, the commit messages, pr message and titles are crucial. These are stored for a long time and should be self explanatory for anyone who decides to go through them. Please write the pr message and commit messages appropriately. You can go through this link for a better understanding of how these are done. 
https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/

Also do sign your commits. You need to have your contributions shown! :)

**Comment by dwaipayan05** [comment]
Closing this ! PR #7 Merged and Accommodated these changes.

**Pull request closed without merging**